### PR TITLE
Add more LA County bus and regional rail operators

### PR DESCRIPTION
**Pull request by DefenestratingWizard**
Added entries for:
- Burbank Bus
- ~~Glendale Beeline~~
- Santa Clarita Transit
- AVTA
- Metrolink

this is my first PR so I may have made a mistake

### CODE REVIEW

File: feeds/us-ca.json [hunk 1]
```diff
@@ -90,6 +90,30 @@
...
         },
+        {
+            "name": "Antelope-Valley-Transit-Authority",
+            "type": "transitland-atlas",
+            "transitland-atlas-id": "f-antelope~valley~transit~authority"
+        },
+        {
+            "name": "Glendale-Beeline",
+            "type": "transitland-atlas",
+            "transitland-atlas-id": "f-9q5f-glendalebeeline"
```
Comment (mlundblad) [inline feeds/us-ca.json]: It looks like this gives a 403 (Permission denied).
Maybe it needs an addional authentication header with an API key, or something like that?

Comment (DefenestratingWizard) [reply feeds/us-ca.json]: it seems to allow download when the feed url is opened in firefox. I can't seem to find information about an API key or required headers (or any info other than the transitland entries) so maybe it makes sense to remove Beeline for now.

File: feeds/us-ca.json [hunk 2]
```diff
@@ -90,6 +90,30 @@
...
+        },
+        {
+            "name": "Burbank-Bus",
```
Comment (mlundblad) [inline feeds/us-ca.json:106]: This one seems to stumble on agency.txt (something with the language code).
Maybe adding:

"fix": true
would resolve this (I'm not entirely sure)

File: feeds/us-ca.json (hunk 1 above)
Comment (jbruechert) [reply feeds/us-ca.json]: The server only seems to answer with the data if the client advertises support for compressed transfers. Apparently we don't do that right now, we should fix that.
If you don't want to do it right now, I suggest adding skip: true with a skip-reason for now so we can add it later.

**Comment by mlundblad** [comment]
Thanks!
Looks good!
(there's a couple of other feeds failing now, but those are unrelated to these)

File: feeds/us-ca.json (hunk 1 above)
Comment (mlundblad) [reply feeds/us-ca.json]: Yeah, it's marked as skipped for now.

**Pull request merged**